tools/depends/target/libcdio/Makefile

```diff
@@ -11,6 +11,22 @@ CONFIGURE=./configure --prefix=$(PREFIX)  --with-cd-drive=no --with-cd-info=no \
           --with-cdda-player=no --with-cd-read=no --with-iso-info=no --with-iso-read=no --disable-example-progs \
           --disable-cpp-progs --enable-cxx --disable-shared
 
+ifeq ($(CROSS_COMPILING), yes)
+  DEPS += ../../Makefile.include
+  CXXFLAGS += -std=c++11
```

Comment:
Why is this now needed for a complete depends build?

Reply:
it's copy/paste from crossguid